internal/pkg/tfconf/output.go

```diff
@@ -1,11 +1,29 @@
 package tfconf
 
+import (
+	"encoding/json"
+)
+
 // Output represents a Terraform output.
 type Output struct {
```

Comment:
By changing the Value attribute's type to `interface{}`, we are able to include typing in the generated printouts.

Reply:
I just opened PR #196 which addresses this (well `Input`, but this is similar). I think it's fine we keep this change of yours (i.e. using `interface{}`) as is here and I'll rebase mine later when we get this PR merged. :+1: